portalnet/src/find/iterators/findcontent.rs

```diff
@@ -56,15 +56,15 @@ pub enum FindContentQueryPending<TNodeId> {
         // peer that sent the content
         peer: TNodeId,
         // channel used to send back the validated content info
-        valid_content_tx: Sender<ValidatedContent<TNodeId>>,
+        valid_content_tx: Sender<Option<ValidatedContent<TNodeId>>>,
```

Comment:
We spoke about how to differentiate between a failed utp and failed validation within the trace, and you mentioned it would be tough to send the trace between threads. I think we could instead pass back a wrapper datatype enum here with `ValidatedContent`, `FailedUtp`, and `FailedValidation` variants instead of `None` on failure. 

Not a blocker on this PR.